packages/jest-circus/src/run.ts

```diff
@@ -38,46 +38,45 @@ const _runTestsForDescribeBlock = async (
 
   if (!isSkipped) {
     for (const hook of beforeAll) {
+      if (describeBlock.errors.length > 0) {
```

Comment:
Why this logic change? I don't have any opinions about the change itself, but seems unrelated to typo fix 🙂

Reply:
@SimenB yup , I am pulling a new one this one seems to have more area of improvement. :-)